java-sdk-logging/java-sdk-logging-bom/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.api</groupId>
+    <artifactId>java-sdk-logging-parent</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
I wonder if we want to keep the version name the same as the main project?

Reply:
A followup to this, if we keep this standalone with its own versions, what version should we start with? 0.1.0 seems to indicate unstable, do we want that?

Reply:
I think 0.1.0 is fine now since we are still developing it.

We can change the version to 1.0.0 when we want to release the library.